### PR TITLE
Updated tag documentation to reflect new state

### PR DESCRIPTION
**Pull request by Jelledb**

### CODE REVIEW

**Comment by Jelledb** [comment]
Added both of you guys as reviewers since I think you both offer a unique perspective on the tag editor.

**Comment by chewbeccy** [comment]
@sttomm @Jelledb was thinking about getting rid of the wording metadata tags. Lets call them meshTags instead to have the same wording on blogs, presentation, website and general communication.

File: docs/meshstack.metadata-tags.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
 ---
 id: meshstack.metadata-tags
-title: Metadata Tags
+title: Configuring Tags
```
Comment (chewbeccy) [inline docs/meshstack.metadata-tags.md:3]:
```suggestion
title: Managing Tags
```

File: docs/meshstack.metadata-tags.md [hunk 2]
```diff
@@ -229,8 +106,8 @@ meshStack makes metadata available to [Landing Zones](./meshcloud.landing-zones.
 meshStack automatically derives [metadata tags](./meshcloud.metadata-tags.md) for [meshTenants](./meshcloud.tenant.md) based on the metadata tags set on the [meshProject](./meshcloud.project.md), the [payment method](./meshcloud.payment-methods.md) configured on the meshProject and
 the [meshCustomer](./meshcloud.customer.md) it belongs to.
 
-It's possible that these sources provide different values for the same tag key. For example, both the meshCustomer and
-meshProject [tag schema](#tag-schemas) could define a `cmdb-id` tag. Setting the `cmdb-id` tag value on the
+It's possible that these objects have tags with the same tag key. For example, both the meshCustomer and
+meshProject could contain a `cmdb-id` tag. Setting the `cmdb-id` tag value on the
 meshCustomer provides it as a "default" value to all tenants owned by that meshCustomer. A user can then override
```
Comment (chewbeccy) [inline docs/meshstack.metadata-tags.md:111]: we should mention here that a project can also inherit tags from a customer

File: docs/meshstack.metadata-tags.md [hunk 3]
```diff
@@ -249,19 +126,18 @@ Some Landing Zone assets like [GCP Cloud Functions](meshstack.gcp.landing-zones.
...
 
 ## Tags in Cloud Tenants
 
-Certain cloud platforms allow you to tag resources in order to help organize them. It is possible to use the metadata tags from meshStack to control how these resources are tagged during project replication. The whole lifecycle of this tags is managed. This means tags on the cloud platform are updated and removed depending on the underlying metadata of the replicated meshProject. In order to properly track meshStack managed tags, every tag is usually prefixt with a tag prefix (e.g. the `meshstack_` prefix in the tag label `meshstack_costcenter:12345`).
+Certain cloud platforms allow you to tag resources in order to help organize them. It is possible to use the metadata tags from meshStack to control how these resources are tagged during project replication. The whole lifecycle of these tags is managed. This means tags on the cloud platform are updated and removed depending on the underlying metadata of the replicated meshProject. In order to properly track meshStack managed tags, every tag is usually prefixed with a tag prefix (e.g. the `meshstack_` prefix in the tag label `meshstack_costcenter:12345`).
```
Comment (chewbeccy) [inline docs/meshstack.metadata-tags.md:137]: we should mention that we dont support resource tags natively but support fully tags on tenants

**Review by chewbeccy — COMMENTED** [review]
-s

**Comment by Jelledb** [comment]
> @sttomm @Jelledb was thinking about getting rid of the wording metadata tags. Lets call them meshTags instead to have the same wording on blogs, presentation, website and general communication.

This would be fantastic for marketing: hashtag meshtag!!!!

File: docs/meshstack.metadata-tags.md [hunk 4]
```diff
@@ -22,168 +25,42 @@ such metadata include:
...
-The schemas need to be available in the configuration repository.
+This page will show you all tags that are currently active on supported meshObjects. If you would like to know more details about a tag, or create, edit, or delete a tag, you can do so by clicking on the large arrow ▶️ on the right for a meshObject.
+This next page will show you more details about a tag such as the title, description or data type of a tag.
 
-### Tags on Payment Methods
+### Defining a new tag
 
-[Payment Methods](meshcloud.payment-methods.md) can also provide tag information. For example,
-the default "cost center" payment method provides the `costCenter` tag.
+You can define a new tag by clicking '+ Create' at the top right in the page mentioned above.
```
Comment (j0g3sc) [inline docs/meshstack.metadata-tags.md:43]: I find this misleading as i connected "the page mentioned above" with the screenshot, but the create button is only available on one of the sub-screens.

File: docs/meshstack.metadata-tags.md [hunk 5]
```diff
@@ -22,168 +25,42 @@ such metadata include:
...
+
+> Hint: after entering a name, you can view the tag input as if it were shown to the end user. You can do this on the right-side of the screen where it says 'Tag Preview'.
+> This can help you get an idea of how it will look when end-users fill in the tag value.
+
+- **Display Name**: This is the text that will be shown above the input field which the user reads. Note that is **not** the name of the tag itself.
+- **Name**: This is the actual name of the tag as it will be displayed in the green bubble. It is also the name that is used for uniquely identifying the tag and when exporting it or when [using it within a landing zone](#exposing-tags-via-landing-zones). This name cannot be changed later (at the moment). We recommend using a name that is similar or the same as the Display Name. Keep in mind that the use of spaces is not allowed since the tag name is used for technical purposes.
+- **Description (optional)**: This is the description of the tag that will be displayed in small grey text above the tag input. It is useful for giving extra information to the user when the (display) name of the tag is not explanatory enough.
```
Comment (j0g3sc) [inline docs/meshstack.metadata-tags.md:52]: although i get what you want to say, I find the `(display)` a bit strange. I would probably just omit it.

File: docs/meshstack.metadata-tags.md [hunk 6]
```diff
@@ -22,168 +25,42 @@ such metadata include:
...
+- **Default Value (optional)**: This can be used to give a default value to the user for contextual help or for filling in a default.
+- **Mandatory**: If a tag is mandatory, it means the user cannot create a new meshObject without first filling in this tag value.
+- **Restricted**: If a tag is restricted, it means only Partner users can enter this tag value. The input will also not be shown to the end-user when creating a new meshObject. This is useful if an organization doesn't want users to edit these values in self-service. For example, an organization can use restricted tags to implement a "segregation of duty" control to ensure that a partner user confirmed a project's data classification.
+- **Replication**: (note: not supported for meshLandingZones) If a tag has replication enabled, it will be used to tag an actual tenant in the cloud platform.
+  - **Replication Tag Key**: Although we recommend using the exact same value as **Name** here, it possible to use a slightly different tag key when it is used for replication. Also, keep in mind that the tag key will be prefixed, depending on what is configured for [tenant tags](#tags-in-cloud-tenants).
```
Comment (j0g3sc) [inline docs/meshstack.metadata-tags.md:59]: d: I think also tag sanitization will be applied for this field right? Maybe we should add a sentence that mentions this, to make users aware, that these could differ in the various clouds.

File: docs/meshstack.metadata-tags.md [hunk 7]
```diff
@@ -22,168 +25,42 @@ such metadata include:
...
+- **Regular Expression (optional)**: This is only available for `e-mail` and `string` tag types and can be used to enforce that users enter a certain data format. Also make sure to mention this in the description to help users what to enter.
+- **Default Value (optional)**: This can be used to give a default value to the user for contextual help or for filling in a default.
+- **Mandatory**: If a tag is mandatory, it means the user cannot create a new meshObject without first filling in this tag value.
+- **Restricted**: If a tag is restricted, it means only Partner users can enter this tag value. The input will also not be shown to the end-user when creating a new meshObject. This is useful if an organization doesn't want users to edit these values in self-service. For example, an organization can use restricted tags to implement a "segregation of duty" control to ensure that a partner user confirmed a project's data classification.
+- **Replication**: (note: not supported for meshLandingZones) If a tag has replication enabled, it will be used to tag an actual tenant in the cloud platform.
```
Comment (j0g3sc) [inline docs/meshstack.metadata-tags.md:58]: d: maybe add a yellow field for the notes as it is done for the hint above.

File: docs/meshstack.metadata-tags.md [hunk 8]
```diff
@@ -229,8 +106,8 @@ meshStack makes metadata available to [Landing Zones](./meshcloud.landing-zones.
 meshStack automatically derives [metadata tags](./meshcloud.metadata-tags.md) for [meshTenants](./meshcloud.tenant.md) based on the metadata tags set on the [meshProject](./meshcloud.project.md), the [payment method](./meshcloud.payment-methods.md) configured on the meshProject and
 the [meshCustomer](./meshcloud.customer.md) it belongs to.
 
-It's possible that these sources provide different values for the same tag key. For example, both the meshCustomer and
-meshProject [tag schema](#tag-schemas) could define a `cmdb-id` tag. Setting the `cmdb-id` tag value on the
+It's possible that these objects have tags with the same tag key. For example, both the meshCustomer and
+meshProject could contain a `cmdb-id` tag. Setting the `cmdb-id` tag value on the
```
Comment (j0g3sc) [inline docs/meshstack.metadata-tags.md:110]: d: I like the example. We could add a table or image or something that shows the different tags at meshObject level and then the tag that is actually used for resource tagging to give a visual example in addition to the precedence rule.
e.g.
meshObject  | tag name | tag value |
meshCustomer |  cmdb-id  |   12         |
meshPaymentMethod |  cmdb-id  |   44         |
meshProject |  cmdb-id  |   79         |
result | cmdb-id | 79 |

**Review by j0g3sc — APPROVED** [review]

**Pull request merged**